tests/network/vmi_multus.go

```diff
@@ -332,26 +353,61 @@ var _ = SIGDescribe("[Serial]Multus", func() {
 		})
 
 		Context("VirtualMachineInstance with Linux bridge plugin interface", func() {
-			table.DescribeTable("should be able to ping between two vms", func(interfaces []v1.Interface, networks []v1.Network, ifaceName string) {
+			retrieveDynamicIPV4AddressOfSecondaryInterface := func(vmi *v1.VirtualMachineInstance) (string, error) {
+				By("retrieving the dynamic ip assigned to the secondary interface from dhcp")
+				vmiPod := tests.GetRunningPodByVirtualMachineInstance(vmi, vmi.Namespace)
+				output, err := tests.ExecuteCommandOnPod(
+					virtClient,
+					vmiPod,
+					"compute",
+					[]string{"ip", "address", "show", "dev", "net1"},
+				)
+				if err != nil {
+					return "", err
+				}
+				r, err := regexp.Compile(`[0-9]+\.[0-9]+\.[0-9]+\.[0-9]/[0-9]+`)
+				if err != nil {
+					return "", err
+				}
+
+				return cidrToIP(r.FindString(output))
+			}
```

Comment:
Can't say I like this part (retrieving the IP via console), but unfortunately, it's either that, or rely on the guest agent (which you'd have to install).

This would never the less be a reason for you to use a fedora based image (we have an img that comes w/ guest agent pre-installed). Should have thought of that before.